### ISSUE TITLE
Windows 10 network provider issue

### ISSUE THREAD

**Issue by guilhermemarconi** [scotch-io/scotch-box#91]
Don't know if here is the best place to report this, but there goes.

Probably everybody here knows that yesterday Microsoft released Windows 10. I did the update and, now, I'm having problems with VirtualBox (probably this one [here (ticket 14040)][1]).

I'm creating this issue because I don't know if something will also have to change in the box to work well with the (possible) update of VirtualBox.

Yesterday, before updating my Windows (my last version was 7), everything was working just fine with Vagrant 1.7.2 + VirtualBox 4.3.23 + Scotch Box, and today I have already updated Vagrant (1.7.4) and VirtualBox (first 4.3.30 and after to 5.0.0) and nothing still works. Even getting better results with VirtualBox 5.0.0, it still not working.

So, if someone here didn't know, it's happening. Today I lost a work day trying to solve this and still can't do anything.

[1]:https://www.virtualbox.org/ticket/14040

**Comment by TheDutchCoder** [comment]
I have the same issue, it seems to be network controller related.
The error I'm getting is:

There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["hostonlyif", "create"]

Stderr: 0%...
Progress state: E_FAIL
VBoxManage.exe: error: Failed to create the host-only adapter
VBoxManage.exe: error: Code E_FAIL (0x80004005) - Unspecified error (extended info not available)
VBoxManage.exe: error: Context: "int __cdecl handleCreate(struct HandlerArg *,int,int *)" at line 66 of file VBoxManageHostonly.cpp

**Comment by rodpater** [comment]
This is an issue with VirtualBox/Vagrant, they are trying to solve this at this moment.
Ref.: https://github.com/mitchellh/vagrant/issues/6059.

**Comment by EdEichman** [comment]
VirtualBox is [NOT working on this](https://www.virtualbox.org/ticket/14040) ("I will also point out that Oracle cannot and will not support Genymotion products (the same goes for docker, vagrant, etc.))".

People using Windows 10 Pro 64 bit / vagrant 1.7.4 and VirtualBox 5.0.1-102010 ARE successfully [running their boxes](https://github.com/mitchellh/vagrant/issues/6059) (see end of link... I've also seen others reporting that it's working).

I'm on Win10 to stay, and would love to stay with Scotch Box, but have tried everything and cannot get it working.

Nick, I know you're busy, but would be great if you could give this a look. I guess I'm not the only guy moving to Win10, and nothing seems to be moving on the vbox / vagrant side.

**Comment by ncerminara** [comment]
Can you get any boxes working? This doesn't seem to be a Scotch Box issue, but a Vagrant and VirtualBox one. Homestead is a more widely used Vagrant setup, you might be better searching in Google getting "Homestead working on Windows 10". It's all the same if that works.

Some things that might help you until updates are pushed:

- https://laracasts.com/discuss/channels/general-discussion/windows-10-vagrant-virtualbox-homestead (specifically towards the end of that forum)
- http://stackoverflow.com/questions/31756684/vagrant-up-after-upgrade-to-windows-10
- https://forums.virtualbox.org/viewtopic.php?f=6&t=67675
- http://iam.hoangstark.com/windows/2015/07/31/get-homestead-working-on-windows-10.html
- Make sure VT-x is enabled (http://www.microsoft.com/en-za/download/details.aspx?id=592)

It seems your best bet is to completely uninstall everything related to it and follow the instructions from one of these resources

**Comment by EdEichman** [comment]
Nick, thanks for looking. I'll work through your list and let you know if I get something working.

**Comment by EdEichman** [comment]
I was not able to get it working. I've moved back to XAMPP for the moment... will switch back to Scotch Box when things have settled.

**Comment by ncerminara** [comment]
Bummer. Did you make sure you completely uninstalled everything before trying the fixes?

Anyway - watch this thread. I'll keep it updated and open as more info comes in. For anyone else looking for answers, there are tons of people who got Vagrant working on Windows 10.

**Comment by EdEichman** [comment]
Yes, I completely uninstalled several times, and I will watch this thread.

Also, I was starting from a 100% clean Windows 10 (moved from 32 bit to 64 bit... only a full install was possible), and who knows... maybe being "too" clean is my problem ;->

But enough for now... I need to code.

Thanks, Nick!

**Comment by mcjohnston** [comment]
I finally got my box working again after following this comment:
https://github.com/mitchellh/vagrant/issues/6068#issuecomment-127233545

Additionally, after doing a clean install of Vagrant ( 1.7.4 ) and VBox (5.0.2) I did a vagrant destroy on the box and left the fix executable open for the full initial vagrant up. 

Hope it helps anyone still having issues.

**Comment by ncerminara** [comment]
Thanks for the tips. I'm sure this will all be resolved soon and is a priority for Hashicorp

**Comment by rodpater** [comment]
I´ve fixed my problem with a test build recommended version of VBox: 
[testbuilds](https://www.virtualbox.org/wiki/Testbuilds)
and closed temporarily my 3rd party firewall app, runs nicely now.

**Comment by ncerminara** [comment]
Nice! Thanks for sharing with everyone

**Comment by kaosmos** [comment]
Hello there,
I'm using Windows 10 and I was a Vagrant/Scotch Box virgin but now I have everything working fine.
I updated Virtual Box to 5.0.6 (since the 4.x I had stopped working in Win10) and I followed the steps detailed in https://box.scotch.io/ and everything went fine, though I had an annoyance regarding the network setup.
The default settings in Vagrantfile starts the machine on an IP outside of my LAN and I can't reach it with my usual network setup, following [this answer on StackOverflow](http://stackoverflow.com/a/23508647/2477547) I modified the Vagrantfile to make the NAT port forwarding and now everything's fine.
I don't see any contraindication in this setup, why you choose a to start the box on an IP other than localhost's one?
